src/Config.php

```diff
@@ -40,7 +40,7 @@ class Config
     /**
      * @var string
      */
-    protected $logger_class = 'EasySwoole\Permission\Logger';
+    protected $logger_class = Logger::class;
```

Comment:
Sorry, I didn't see this `pr`.
It's not reasonable to write `classname` in this place, but now that the version has been released, let's do it.